<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-10T16:20:07 IST" name="Default suite" finished-at="2022-12-10T16:20:15 IST" duration-ms="7992">
    <groups>
    </groups>
    <test started-at="2022-12-10T16:20:07 IST" name="Default test" finished-at="2022-12-10T16:20:15 IST" duration-ms="7992">
      <class name="webAuto.Prescription">
        <test-method is-config="true" signature="initialization()[pri:0, instance:webAuto.Prescription@57a3af25]" started-at="2022-12-10T16:20:07 IST" name="initialization" finished-at="2022-12-10T16:20:14 IST" duration-ms="6176" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:webAuto.Prescription@57a3af25]" started-at="2022-12-10T16:20:14 IST" name="setUp" finished-at="2022-12-10T16:20:14 IST" duration-ms="93" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginAsDoctorInvalidValue()[pri:0, instance:webAuto.Prescription@57a3af25]" started-at="2022-12-10T16:20:14 IST" name="loginAsDoctorInvalidValue" finished-at="2022-12-10T16:20:15 IST" duration-ms="1188" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please Enter 10 Digit of Phone Number] but found [Please Enter 10 Digit of Phone]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please Enter 10 Digit of Phone Number] but found [Please Enter 10 Digit of Phone]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at webAuto.Prescription.loginAsDoctorInvalidValue(Prescription.java:157)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsDoctorInvalidValue -->
        <test-method is-config="true" signature="screenShot(org.testng.ITestResult)[pri:0, instance:webAuto.Prescription@57a3af25]" started-at="2022-12-10T16:20:15 IST" name="screenShot" finished-at="2022-12-10T16:20:15 IST" duration-ms="433" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginAsDoctorInvalidValue status=FAILURE method=Prescription.loginAsDoctorInvalidValue()[pri:0, instance:webAuto.Prescription@57a3af25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenShot -->
      </class> <!-- webAuto.Prescription -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
